tests/e2e/login/privacy-policy.spec.js

```diff
@@ -76,109 +75,95 @@ describe('Privacy policy', () => {
     }
   };
 
-  const closeTourModal = () => {
-    helper.waitElementToBeVisible(element(by.css('.modal-dialog .modal-footer .btn')));
-    element(by.css('.modal-dialog .modal-footer .btn')).click();
+  const closeTourModal = async () => {
+    await helper.waitElementToBeVisibleNative(element(by.css('.modal-dialog .modal-footer .btn')));
```

Comment:
Sorry I missed this one. We have a `utils.closeTour()` can that function replace this?

Reply:
Sure, works fine